buildbot.mariadb.org/master-libvirt/master.cfg

```diff
@@ -235,88 +187,101 @@ f_rpm_upgrade.addStep(getScript('rpm-upgrade.sh'))
-    util.BuilderConfig(name="amd64-ubuntu-1804-deb-autobake-minor-upgrade",
-      workernames=["bb-hz-bbw5-ubuntu-1804"],
-      tags=["Ubuntu", "deb", "upgrade", "kvm"],
-      collapseRequests=True,
-      nextBuild=nextBuild,
-      canStartBuild=canStartBuild,
-      properties={'systemdCapability': 'yes', 'needsGalera': 'yes', 'dist_name': 'ubuntu', 'version_name': 'bionic', 'arch': 'amd64', 'test_mode': 'all', "test_type": "minor"},
-      factory=f_deb_minor_upgrade))
-
-c['builders'].append(
-    util.BuilderConfig(name="amd64-centos-7-rpm-autobake-install",
-      workernames=["buildbot-centos7"],
-      tags=["Centos", "rpm", "install"],
-      collapseRequests=True,
-      nextBuild=nextBuild,
-      canStartBuild=canStartBuild,
-      properties={'systemdCapability': 'yes', 'needsGalera': 'yes', 'version_name': '7', 'arch': 'centos74-amd64'},
-      factory=f_rpm_install))
-
-c['builders'].append(
-    util.BuilderConfig(name="amd64-centos-7-rpm-autobake-major-upgrade",
-      workernames=["buildbot-centos7"],
-      tags=["Centos", "rpm", "upgrade"],
-      collapseRequests=True,
-      nextBuild=nextBuild,
-      canStartBuild=canStartBuild,
-      properties={'systemdCapability': 'yes', 'needsGalera': 'yes', 'version_name': 'centos7', 'arch': 'amd64', "test_type": "major", "test_mode": "server"},
-      factory=f_rpm_upgrade))
-
-c['builders'].append(
-    util.BuilderConfig(name="amd64-centos-7-rpm-autobake-minor-upgrade",
-      workernames=["buildbot-centos7"],
-      tags=["Centos", "rpm", "upgrade"],
-      collapseRequests=True,
-      nextBuild=nextBuild,
-      canStartBuild=canStartBuild,
-      properties={'systemdCapability': 'yes', 'needsGalera': 'yes', 'version_name': 'centos7', 'arch': 'amd64', "test_type": "minor", "test_mode": "server"},
-      factory=f_rpm_upgrade))
-
-c['builders'].append(
-    util.BuilderConfig(name="amd64-fedora-35-rpm-autobake-install",
-      workernames=["bb-hz-bbw5-fedora-35"],
-      tags=["Fedora", "rpm", "install"],
-      collapseRequests=True,
-      nextBuild=nextBuild,
-      canStartBuild=canStartBuild,
-      properties={'systemdCapability': 'yes', 'needsGalera': 'yes', 'version_name': '35', 'arch': 'fedora35-amd64'},
-      factory=f_rpm_install))
+# Add the workers and builds based on the configured install builders (see constants.py)
+for builder_name in builders_install:
+    # Parse builder name
+    platform, os_name, os_version, builder_type = builder_name.split('-')[:4]
+
+    assert builder_type in ['rpm', 'deb']
+    assert platform in ['amd64', 'aarch64']
+
+    os_info_name = os_name + '-' + os_version
+    if platform == 'amd64':
+        libvirt_worker_name = 'bb-hz-bbw5-' + os_name + '-' + os_version
+        connection_url = 'qemu+ssh://buildbot@100.64.100.20:65001/system'
+    elif platform == 'aarch64':
+        libvirt_worker_name = 'bb-eq-arm1-' + os_name + '-' + os_version
+        connection_url = 'qemu+ssh://buildbot@100.64.100.17/system'
```

Comment:
I would love to see those in variables, maybe in a `builder_info.yaml`?
Is it possible? (hostname and IP and port)

Reply:
It should be possible. However not sure if we should keep that info in a separate .yaml file or use the already exiting master-private.cfg

Reply:
My idea behind this is because I want to be able to easily setup a dev environment and that's easier if we have variables for this...

Reply:
I have changed to load the info from master-private.cfg